docker_registry/toolkit.py

```diff
@@ -299,14 +300,23 @@ def exclusive_lock(f):
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         lock_path = os.path.join(
-            '/var/lock', 'registry.{0}.lock'.format(f.func_name)
```

Comment:
This is a hack to hide the fact the current SQL_ALCHEMY code is racy, so, I would rather avoid making this a first class configurable thing.